api/models/comments.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../../db/connection");
+const format = require("pg-format");
+
+exports.selectComments = (article_id) => {
+	article_id
+		? (whereArticleId = `WHERE article_id = ${article_id}`)
+		: (whereArticleId = "");
```

Comment:
Rather than checking if there's an article ID, we should always be passing one into the query. If there isn't an article ID, the SQL query will always fail, as there'll be no comments to get. Please amend the code so that an article_ID is always passed in. You'll need to change your controller as well 👍